tpm2/update_test.go

```diff
@@ -1,153 +0,0 @@
-	c.Check(err, IsNil)
-
-	_, err = s.TPM().PCREvent(s.TPM().PCRHandleContext(23), []byte("foo"), nil)
-	c.Check(err, IsNil)
-	_, _, err = k.UnsealFromTPM(s.TPM())
-	c.Check(err, ErrorMatches, "invalid key data: cannot complete authorization policy assertions: cannot execute PCR assertions: "+
-		"cannot execute PolicyOR assertions: current session digest not found in policy data")
-}
-
-func (s *updateSuite) TestUpdatePCRProtectionPolicyWithPCRPolicyCounter(c *C) {
-	s.testUpdatePCRProtectionPolicy(c, &KeyCreationParams{PCRPolicyCounterHandle: s.NextAvailableHandle(c, 0x01810000)})
-}
-
-func (s *updateSuite) TestUpdatePCRProtectionPolicyNoPCRPolicyCounter(c *C) {
-	s.testUpdatePCRProtectionPolicy(c, &KeyCreationParams{PCRPolicyCounterHandle: tpm2.HandleNull})
-}
-
-func (s *updateSuite) TestUpdatePCRProtectionPolicyWithProvidedAuthKey(c *C) {
-	authKey, err := ecdsa.GenerateKey(elliptic.P256(), testutil.RandReader)
-	c.Check(err, IsNil)
-
-	s.testUpdatePCRProtectionPolicy(c, &KeyCreationParams{
-		PCRPolicyCounterHandle: s.NextAvailableHandle(c, 0x01810000),
-		AuthKey:                authKey})
-}
-
-func (s *updateSuite) testRevokeOldPCRProtectionPolicies(c *C, params *KeyCreationParams) error {
-	key := make(secboot.DiskUnlockKey, 32)
-	rand.Read(key)
-
-	dir := c.MkDir()
-	path := filepath.Join(dir, "key")
-
-	authKey, err := SealKeyToTPM(s.TPM(), key, path, params)
-	c.Check(err, IsNil)
-
-	k2, err := ReadSealedKeyObjectFromFile(path)
-	c.Assert(err, IsNil)
-
-	c.Check(k2.UpdatePCRProtectionPolicy(s.TPM(), authKey, params.PCRProfile), IsNil)
-
-	k1, err := ReadSealedKeyObjectFromFile(path)
-	c.Assert(err, IsNil)
-
-	_, _, err = k1.UnsealFromTPM(s.TPM())
-	c.Check(err, IsNil)
-	_, _, err = k2.UnsealFromTPM(s.TPM())
-	c.Check(err, IsNil)
-
-	c.Check(k1.RevokeOldPCRProtectionPolicies(s.TPM(), authKey), IsNil)
-
-	_, _, err = k1.UnsealFromTPM(s.TPM())
-	c.Check(err, IsNil)
-	_, _, err = k2.UnsealFromTPM(s.TPM())
-	c.Check(err, IsNil)
-
-	c.Check(k2.RevokeOldPCRProtectionPolicies(s.TPM(), authKey), IsNil)
-
-	_, _, err = k2.UnsealFromTPM(s.TPM())
-	c.Check(err, IsNil)
-	_, _, err = k1.UnsealFromTPM(s.TPM())
-	return err
-}
-
-func (s *updateSuite) TestRevokeOldPCRProtectionPoliciesWithPCRPolicyCounter(c *C) {
-	err := s.testRevokeOldPCRProtectionPolicies(c, &KeyCreationParams{
-		PCRProfile:             tpm2test.NewPCRProfileFromCurrentValues(tpm2.HashAlgorithmSHA256, []int{7, 23}),
-		PCRPolicyCounterHandle: s.NextAvailableHandle(c, 0x01810000)})
-	c.Check(err, ErrorMatches, "invalid key data: cannot complete authorization policy assertions: the PCR policy has been revoked")
-}
-
-func (s *updateSuite) TestRevokeOldPCRProtectionPoliciesWithoutPCRPolicyCounter(c *C) {
-	err := s.testRevokeOldPCRProtectionPolicies(c, &KeyCreationParams{
-		PCRProfile:             tpm2test.NewPCRProfileFromCurrentValues(tpm2.HashAlgorithmSHA256, []int{7, 23}),
-		PCRPolicyCounterHandle: tpm2.HandleNull})
-	c.Check(err, IsNil)
-}
```

Comment:
what happened to these tests?

Reply:
Ah, I missed those when I was moving things around. I've added them back again.